changelog.txt

```diff
@@ -3,6 +3,7 @@ V.Next
 - [MINOR] Add null cursor BrokerCommunicationException error category (#2459)
 - [MINOR] Classifying more UNKNOWN_ERRORs (#2460)
 - [MINOR] Setting flights to control UrlConnection timeout values (#2473)
+- [PATCH] Fix authority data being persisted across different SDK instances (#2462)
```

Comment:
can we update this to reflect that we fixed it only in native auth flows?

Reply:
Done